dockstore-webservice/src/main/resources/migrations.1.11.0.xml

```diff
@@ -102,7 +102,7 @@
                 <constraints nullable="false" primaryKey="true" primaryKeyName="entry_input_fileformat_pkey"/>
             </column>
         </createTable>
-        <addForeignKeyConstraint baseColumnNames="fileformatid" baseTableName="entry_output_fileformat" constraintName="fk_entry_ouput_fileformat" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="fileformat"/>
```

Comment:
Unfortunately, we can't change these migrations that are in the "past"
(Not sure what will happen, we either end up with duplicate constraints in production or maybe it just gets ignored and then prod deviates from qa, neither are good anyway)

Reply:
I have undone the changes